src/wp-content/themes/twentyfourteen/functions.php

```diff
@@ -491,15 +491,24 @@ function twentyfourteen_the_attached_image() {
 	 * @since Twenty Fourteen 1.0
 	 */
 	function twentyfourteen_list_authors() {
-		$contributor_ids = get_users(
-			array(
-				'fields'  => 'ID',
-				'orderby' => 'post_count',
-				'order'   => 'DESC',
-				'who'     => 'authors',
-			)
+		$args = array(
+			'fields'     => 'ID',
+			'orderby'    => 'post_count',
+			'order'      => 'DESC',
+			'capability' => array( 'edit_posts' ),
 		);
 
+		/**
+		 * Filters query arguments for listing authors.
+		 *
+		 * @since 5.9.0
+		 *
+		 * @param array $args Query arguments.
+		 */
+		$args = apply_filters( 'twentyfourteen_list_authors_query_args', $args );
```

Comment:
Not really needed, but figured that with this change to the query args one might want to filter them to assemble this list differently.